### PR TITLE
Flake8 Cleanup

### PR DESCRIPTION
**Pull request by ErikBjorge**
- Remove unused imports
- Improve spacing of functions/methods for readability
- Indentation updates
- Exception expression updates

Signed-off-by: Erik Bjorge <erik.c.bjorge@intel.com>

### CODE REVIEW

**Comment by ErikBjorge** [comment]
To help with reviewing this change I would suggest two passes.  The first with the diff configured to ignore whitespace changes and then a second pass with whitespace changes visible.

File: setup.py [hunk 1]
```diff
@@ -166,17 +169,20 @@ def _build_win_compression(self):
     def run(self):
         # First, we build the standard extensions.
         _build_ext.run(self)
+
         # Then, we build the compression tools and the driver if required
-        driver_build_function = lambda *args: None
```
Comment (npmitche) [inline setup.py:170]: Are lambda functions against flake8?

File: chipsec_main.py [hunk 2]
```diff
@@ -401,41 +423,49 @@ def parse_args(self):
...
+                                                                           self._cs.pch_did, self._cs.pch_rid)
+        ret["Version"] = "{}".format(self.version)
         ret["Message"] = "{}".format(self.message)
         return ret
 
-    def log_properties( self ):
-        logger().log("[CHIPSEC] OS      : {} {} {} {}".format(self._cs.helper.os_system, self._cs.helper.os_release, self._cs.helper.os_version, self._cs.helper.os_machine) )
-        logger().log("[CHIPSEC] Python  : {} ({})".format(platform.python_version(), "64-bit" if self.PYTHON_64_BITS else "32-bit"))
+    def log_properties(self):
+        chip_info = "[CHIPSEC] {:8}: {}\n[CHIPSEC]      VID: {:04X}\n" \
```
[the remark this answers is not in the document]
Comment (ErikBjorge) [reply chipsec_main.py:458]: Sure, not a problem at all.  I will take care of that.

File: chipsec_util.py [hunk 3]
```diff
@@ -180,12 +192,15 @@ def main(self):
                 sys.exit(ExitCode.OK)
 
         if self._show_banner:
+            chip_info = "[CHIPSEC] {:8}: {}\n[CHIPSEC]      VID: {:04X}\n" \
```
[the remark this answers is not in the document]
Comment (ErikBjorge) [reply chipsec_util.py:195]: Same answer as above

File: setup.py (hunk 1 above)
Comment (ErikBjorge) [reply setup.py:170]: Assignment of lambda functions is what Flake 8 is complaining about.  They are really meant to be used inline.

**Review by frinzell — APPROVED** [review]

**Comment by kerneis-anssi** [comment]
If we're ready to have cleaners run and reformat existing code, what about switching to fully-automated code formatting, enforced in presubmit checks? I used to appreciate it at Google, removes a lot of cognitive burden. Google uses yapf but Black seems more popular for open-source projects: https://github.com/psf/black

**Comment by ErikBjorge** [comment]
Thanks for the pointer to the project. Using a tool like Black is definitely worth investigating.

**Comment by kerneis-anssi** [comment]
I'd be happy to give it a go later this week and make a sample pull request (separating tooling and reformatting of course) if I can make it work. Actually rolling it out would require some synchronization, eg. just before or after a release. I'll attend the community meeting tomorrow.

**Comment by npmitche** [comment]
We are planning on doing a release today. (1.8.2) Lets discuss this in the meeting tomorrow, but the timing should work pretty well.

**Pull request merged**